queries.json

```
@@ -47,19 +47,34 @@
       "description": "Enumerates the classful networks for all destination IP addresses including count of connections"
     },
     {
-      "name": "Suricata Alerts by Category",
+      "name": "Suricata Alerts by Severity & Category",
       "value": "event_type==\"alert\" | count() by alert.severity,alert.category | sort count",
-      "description": "Shows all Suricata alert counts by category and severity"
+      "description": "Shows all Suricata alert counts, grouped by category and severity"
     },
     {
-      "name": "Suricata Alerts by Source and Destination",
+      "name": "Suricata Alerts by Signature",
+      "value": "event_type==\"alert\" | count() by alert.signature | sort count",
+      "description": "Shows all Suricata alert counts, grouped by signature"
+    },
+    {
+      "name": "Suricata Alert Categories by Source and Destination",
       "value": "event_type==\"alert\" | alerts := union(alert.category) by src_ip, dest_ip",
-      "description": "Shows all Suricata alerts in a list by unique source and destination IP addresses"
+      "description": "Shows categories of all Suricata alerts in a list, grouped by unique source and destination IP addresses"
```

Nit: Make the structure here more consistent with that of the other descriptions.
```suggestion
      "description": "Shows a list of Suricata alert categories, grouped by unique source and destination IP addresses"
```